app-connector-client/server/routes/events.js

```diff
@@ -48,11 +49,12 @@ async function createEventsFromConfig(localKyma, eventsConfig, registeredApis) {
 
         } catch (error) {
             LOGGER.error("Registration of Event API '%s' failed: %s", event.name, JSON.stringify(error))
+            throw error

```

Comment:
We should collect the errors at auto-register and throw an overall error at the end

Reply:
should this be done in this PR

Reply:
Done